home/home2.1/test_script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+g++ -o main main.cpp
+
+run_test() {
+    local expected="$1"
+    shift
+    local output=$(./main "$@")
+    if [ "$output" == "$expected" ]; then
+        echo "Test passed: $@"
+    else
+        echo "Test failed: $@"
+        echo "Expected: $expected"
+        echo "Got: $output"
+    fi
+}
+
+clear_history() {
+    ./main "bread"
+}
+
+clear_history
+echo "Running Test Case 1"
+run_test "Welcome, Anastasiia!" "Anastasiia"
+run_test "Welcome, Gabriele!" "Gabriele"
+run_test "Hello again(x=2), Gabriele" "Gabriele"
+run_test "Hello again(x=2), Anastasiia" "Anastasiia"
+run_test "Hello again(x=3), Gabriele" "Gabriele"
+run_test "Welcome, Markel!" "Markel"
+clear_history
```

Comment:
Formatting hope it should be moved below as this is a part of the next test case not this one.